t/response.t

```diff
@@ -183,3 +183,15 @@ is($r2->filename(), 'baz.txt');
 $r2->remove_header ('Content-Location');
 $req->uri('http://www.example.com/bar.txt');
 is($r2->filename(), 'bar.txt');
+
+# RFC 8187
+## Encoding: See <http://purl.org/NET/http/content-disposition-tests#encoding-2231-char>.
+$r2->header ('Content-Disposition' => "attachment; filename*=iso-8859-1''foo-%E4.html\n");
+is($r2->filename(), 'foo-ä.html', 'attwithisofn2231iso');
```

Comment:
These test labels are very terse - could you make them longer and more readable?